.github/workflows/github-cxx-qt-tests.yml

```diff
@@ -224,6 +224,9 @@ jobs:
           - name: Ubuntu 22.04 (gcc) Qt5
             os: ubuntu-22.04
             qt_version: 5
+            aqt_version: '5.15.2'
+            arch: 'gcc_64'
+            host: 'linux'
```

Comment:
i would maybe prefix all of these with `aqt` so `aqt_host` and `aqt_arch` etc to avoid any confusion or leaking into setting other things